corehq/apps/app_manager/util.py

```diff
@@ -580,7 +580,9 @@ def app(self):
 
     def clear_caches(self):
         self.get_latest_app_version.clear(self)
+        self.get_latest_apk_version.clear(self)
 
+    @quickcache(vary_on=['self.app_id'])
```

Comment:
Looks good, I am not sure if there was an explicit reason we didn't cache this originally when we added this but cached `get_latest_app_version`.